Java/PoolAndResourceFile/src/main/java/PoolAndResourceFile.java

```diff
@@ -0,0 +1,297 @@
+import java.io.*;
+import java.net.URISyntaxException;
+import java.security.InvalidKeyException;
+import java.util.*;
+import java.util.concurrent.TimeoutException;
+
+import com.microsoft.azure.storage.*;
+import com.microsoft.azure.storage.blob.*;
+import org.apache.commons.io.IOUtils;
+
+import com.microsoft.azure.batch.*;
+import com.microsoft.azure.batch.auth.BatchSharedKeyCredentials;
+import com.microsoft.azure.batch.protocol.models.*;
+
+public class PoolAndResourceFile {
+
+    // Create IaaS pool if pool isn't exist
+    private static CloudPool CreatePool(BatchClient client, String poolId) throws BatchErrorException, IllegalArgumentException, IOException, InterruptedException, TimeoutException {
+        // Pool only have 3 D1 VM
+        String osPublisher = "OpenLogic";
+        String osOffer = "CentOS";
+        String poolVMSize = "STANDARD_D1";
+        int poolVMCount = 1;
+        long POOL_STEADY_TIMEOUT = 5 * 60 * 1000;
+        long VM_READY_TIMEOUT = 20 * 60 * 1000;
+
```

Comment:
nitpick: Add comments around how long these times are. Using the Duration class may be a better alternative in this case: https://docs.oracle.com/javase/8/docs/api/java/time/Duration.html